uniffi_bindgen/src/bindings/kotlin/gen_kotlin/custom.rs

```diff
@@ -2,39 +2,23 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-use super::{filters, CustomTypeConfig};
-use crate::backend::{CodeDeclaration, CodeOracle, CodeType, Literal};
-use crate::interface::{FFIType, Type};
-use askama::Template;
-use std::borrow::Borrow;
+use crate::backend::{CodeOracle, CodeType, Literal};
 
 pub struct CustomCodeType {
     name: String,
-    builtin: Type,
-    config: Option<CustomTypeConfig>,
 }
 
 impl CustomCodeType {
-    pub fn new(name: String, builtin: Type, config: Option<CustomTypeConfig>) -> Self {
+    pub fn new(name: String) -> Self {
         CustomCodeType {
             name,
-            builtin,
-            config,
         }
     }
 }
 
 impl CodeType for CustomCodeType {
-    fn type_label(&self, oracle: &dyn CodeOracle) -> String {
-        match &self.config {
-            // We have a custom type config use the supplied type name from the config
-            Some(custom_type_config) => custom_type_config
-                .type_name
-                .clone()
-                .unwrap_or_else(|| self.name.clone()),
-            // No custom type config, use our builtin type
-            None => self.builtin.type_label(oracle),
-        }
+    fn type_label(&self, _oracle: &dyn CodeOracle) -> String {
+        self.name.clone()
```

Comment:
In general I didn't change the generated code, just how we were generating it.  This is the closest thing to a functional change.  When we're rending the type name for a custom type, we always use the name from the UDL.  Instead of sometimes using the bultin type name or the concrete type name, we make a typealias in the template code.